packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -103,26 +114,43 @@ export function viewFromState(
 	initialTree: TreeContent<typeof fuzzSchema.rootFieldSchema>["initialTree"] = undefined,
 ): FuzzView {
 	state.view ??= new Map();
-
-	return (
+	const view =
 		state.transactionViews?.get(client.channel) ??
 		getOrCreate(state.view, client.channel as SharedTree, (tree) => {
+			const treeSchema = treeSchemaFromStoredSchema(
+				(client.channel as SharedTree).storedSchema,
```

Comment:
you should dedupe the casts to `SharedTree` here.

Reply:
Updated `client.channel as SharedTree` to `tree` to dedupe the `SharedTree` casts